backend/participant/utils.py

```diff
@@ -110,8 +108,7 @@ def get_or_create_participant(request):
         participant = Participant(country_code=country_code, access_info=access_info, participant_id_url=participant_id_url)
         participant.save()
         set_participant(request, participant)
-
-    return participant
+        return participant
```

Comment:
With the added tab, I think this will result in only returning the participant in the except clause, right?

```suggestion
    return participant
```

Reply:
Yes, but otherwise, participant is not defined anyway, so this would throw an exception in any case.

Reply:
Ah, I see. It does return participant in the try part already. nvm

Reply:
There is one uncovered possibility though, for which I shall give a suggestion in the next comment

Reply:
Update: I've tested it and it seems to be defined in all cases. So you can either choose the approach of returning `participant` in the end, or by returning `participant` in every clause.

<img width="596" alt="image" src="https://github.com/Amsterdam-Music-Lab/MUSCLE/assets/8208970/a03e27a7-509b-480c-92d8-2d6702b979cd">